src/Oscar.jl

```diff
@@ -39,21 +39,11 @@ if Sys.iswindows()
 end
 
 function _print_banner()
-  println(" -----    -----    -----      -      -----   ")
-  println("|     |  |     |  |     |    | |    |     |  ")
-  println("|     |  |        |         |   |   |     |  ")
-  println("|     |   -----   |        |     |  |-----   ")
-  println("|     |        |  |        |-----|  |   |    ")
-  println("|     |  |     |  |     |  |     |  |    |   ")
-  println(" -----    -----    -----   -     -  -     -  ")
-  println()
-  println("...combining (and extending) ANTIC, GAP, Polymake and Singular")
-  print("Version")
-  printstyled(" $VERSION_NUMBER ", color = :green)
-  print("... \n ... which comes with absolutely no warranty whatsoever")
-  println()
-  println("Type: '?Oscar' for more information")
-  println("(c) 2019-2024 by The OSCAR Development Team")
+  println(raw"  ___  ____   ____    _    _.__       ")
```

Comment:
Shouldn't the first line also have the `|` if you want to match Julia? And no trailing whitespace?
```suggestion
  println(raw"  ___  ____   ____    _    _.__    |")
```

Reply:
Actually, the first line in the Julia banner just consists of the underscore character! To match it, I have made that the first line in the OSCAR banner consists of only underscore characters